schema.ts

```diff
@@ -725,6 +735,12 @@ export interface ImageSchema extends BaseStringSchema<ValueHintType.ImageReferen
   imageOutline?: ImageOutline;
   /** ImageCornerStyle type specifying style of corners on images. If unspecified, default is Rounded. */
   imageCornerStyle?: ImageCornerStyle;
+  /** ImageShapeStyle type specifying shape of image. If unspecified, default is Auto. */
+  imageShapeStyle?: ImageShapeStyle;
+  /** How wide to render the image. Use 0 for default. */
+  width?: number | string;
```

Comment:
What is the `string` option for?

Reply:
yeah, good to add a bit more documentation here.

Reply:
We also support formula values. I was referencing the slider `minimum` and `maximum` properties, which also take number or formula. Should I update that, too?